pkgs/development/python-modules/funcsigs/default.nix

```diff
@@ -9,6 +9,10 @@ buildPythonPackage rec {
     inherit pname version;
     sha256 = "0l4g5818ffyfmfs1a924811azhjj8ax9xd1cffr1mzd3ycn0zfx7";
   };
+  
+  patches = [
+    ./funcsigs-1.0.2-fix-pypy3-tests.patch
```

Comment:
Let's use `fetchpatch` to download the patch from https://raw.githubusercontent.com/gentoo/gentoo/fd9a5a0a3c15d7400b936ce0f0814fd4078e900c/dev-python/funcsigs/files/funcsigs-1.0.2-fix-pypy3-tests.patch

Reply:
How do I do that?

Reply:
There is a section on this in the manual: https://nixos.org/nixpkgs/manual/#sec-patches